usaspending_api/search/v2/views/spending_by_award.py

```diff
@@ -383,9 +412,15 @@ def query_elasticsearch_awards(self) -> Response:
                     {"filter": {"terms": {"covid_spending_by_defc.defc": self.filters.get("def_codes", [])}}}
                 )
             sorts.extend([{field: self.pagination["sort_order"]} for field in sort_field])
+        elif self.pagination["sort_key"] == "Recipient Location" or self.pagination["sort_key"] == "Assistance Listing":
+            sorts = {}
+            for field in sort_field:
+                if field.__contains__("recipient_location_address") or field == "cfda_title.keyword":
```

Comment:
Any reason we can't use `in` instead of `__contains__` here?

```suggestion
                if "recipient_location_address" in field or field == "cfda_title.keyword":
```